core/src/empty/empty.tsx

```diff
@@ -0,0 +1,30 @@
+import s from "./empty.module.css";
+import { createElement } from "react";
+import { Button } from "../button/button";
+import cloudImg from "./empty1.jpg";
+
+interface EmptyProps {
+    active: boolean,
```

Comment:
I think this is unnecessary. We can depend on the definition of the action handler actually: if an action is defined, we show the button, else we dont